x-pack/plugins/osquery/cypress/tasks/saved_queries.ts

```diff
@@ -72,8 +74,7 @@ export const getSavedQueriesComplexTest = () =>
         cy.getBySel(RESULTS_TABLE_BUTTON).click();
 
         // sorting
-        cy.getBySel('dataGridHeaderCell-osquery.egid').click();
-
+        cy.getBySel('dataGridHeaderCellActionButton-osquery.egid').click({ force: true });
```

Comment:
Why do we need `force` ?

Reply:
Cypress was being really flaky about these clicks (I'm not totally sure why as they work just fine in production, I think it's either layout settling or rerenders due to how fast it "clicks" things). CI was consistently failing for me without the `force: true`. If you can get it working in the future with another approach (e.g. a wait or a real click?) feel free to refactor this in the future!